corehq/apps/app_manager/static/app_manager/js/releases/app_diff.js

```diff
@@ -88,9 +88,13 @@ hqDefine('app_manager/js/releases/app_diff', function () {
             ).done(function(appDataOneJson, appDataTwoJson) {
                 try {
                     self.$el.html(self.generateHtmlDiff(appDataOneJson, appDataTwoJson));
-                    self.$el.find('.diff-questions:not(:has(.diff-change))').html(
-                        '<i>' + gettext('No changes detected') + '</i>'
-                    );
+                    // Hide headings with no changes
```

Comment:
Is it possible to not create this markup rather than hiding it, if no one's ever going to see it? It looks like a matter of updating [this loop](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/static/app_manager/js/releases/app_diff.js#L130-L140) to not append sections without changes, but I don't know this code well and might be missing something.

Reply:
got it, thanks for explaining